internal/settings/settings.go

```diff
@@ -25,7 +25,7 @@ func Init() {
 	setConstants()
 
 	viper.AddConfigPath(viper.GetString("config_file"))
-	viper.SetConfigName("config")
```

Comment:
++ to using more descriptive naming. But since it's technically breaking, I think it should be called out in the PR description.